gui/gui.go

```diff
@@ -71,26 +69,19 @@ type Config struct {
 	// AddPaymentRequest creates a payment request from the provided account
 	// if not already requested.
 	AddPaymentRequest func(addr string) error
-	// FetchMinedWork returns all confirmed blocks mined by the pool.
+	// FetchMinedWork returns all blocks mined by the pool.
 	FetchMinedWork func() ([]*pool.AcceptedWork, error)
 	// FetchWorkQuotas returns the reward distribution to pool accounts
 	// based on work contributed per the payment scheme used by the pool.
 	FetchWorkQuotas func() ([]*pool.Quota, error)
-	// FetchPoolHashRate returns the hash rate of the pool.
-	FetchPoolHashRate func() (*big.Rat, map[string][]*pool.ClientInfo)
 	// BackupDB streams a backup of the database over an http response.
 	BackupDB func(w http.ResponseWriter) error
-	// FetchClientInfo returns connection details about all pool clients.
-	FetchClientInfo func() map[string][]*pool.ClientInfo
+	// FetchClientInfo returns details about all connected pool clients.
+	FetchClientInfo func() []*pool.Client
```

Comment:
This would have to be updated as well as the description if you make the changes suggested to the function signature.